lib/DateTimePickerBase/DateTimePickerBase.js

```diff
@@ -161,9 +168,9 @@ module.exports = kind(
 		{name: 'headerWrapper', kind: Item, classes: 'moon-date-picker-header-wrapper', onSpotlightFocus: 'headerFocus', ontap: 'expandContract', components: [
 			// headerContainer required to avoid bad scrollWidth returned in RTL for certain text widths (webkit bug)
 			{name: 'headerContainer', kind: Control, classes: 'moon-expandable-list-item-header moon-expandable-datetime-header', components: [
-				{name: 'header', kind: MarqueeText}
+				{name: 'header', kind: MarqueeText, accessibilityDisabled: true}
```

Comment:
The latest (master #879 or later) blink will ignore `aria-disabled` even if we set `aria-labelledby` on the parent. The related Jira ticket is https://jira2.lgsvl.com/browse/ENYO-2255